testing/scalatest.bzl

```diff
@@ -0,0 +1,6 @@
+def scalatest_repositories():
+    # currently ScalaTest dependencies are already loaded via //scala:scala.bzl#scala_repositories()
+    pass
+
+def scalatest_toolchain():
+    native.register_toolchain("//testing:scalatest_toolchain")
```

Comment:
@liucijus Is this supposed to be `register_toolchains`? 
Added the following in my WORKSPACE file:
```
load("@io_bazel_rules_scala//testing:scalatest.bzl", "scalatest_toolchain")

scalatest_toolchain()
```
and it errored out with:
```
Error: no native function or rule 'register_toolchain'
```

Reply:
Thanks for reporting, here's the fix: https://github.com/bazelbuild/rules_scala/pull/1130